### PR TITLE
{tools}[GCCcore/11.2.0] binutils v2.37, GCCcore v11.2.0, zlib v1.2.11, ...

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)
Maybe also want to include patch for https://gcc.gnu.org/bugzilla/show_bug.cgi?id=101683 once it's done?

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/27ce1da1701b76bbc2bfbf910b8fff2a for a full test report.

File: easybuild/easyconfigs/f/flex/flex-2.6.4-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+source_urls = ['https://github.com/westes/flex/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995']
+
+builddependencies = [
+    ('Bison', '3.7.6'),
+    ('help2man', '1.48.3'),
+    # use same binutils version that was used when building GCC toolchain
+    ('binutils', '2.36.1', '', True),
```
Comment (branfosj) [inline easybuild/easyconfigs/f/flex/flex-2.6.4-GCCcore-11.2.0.eb:23]:
```suggestion
    ('binutils', '2.37', '', True),
```

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/9e8c147467ef6639f0811a74f1eea414 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/8d860c0dd0c6b187c2b833b8b5c5b841 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in total)
bber0501u03a.bb2.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/ca7e5bbbbb0478b39cc3796060bd1f8e for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/22dce90a691921653d8c7d989b73be4d for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in total)
bear-pg0305u19a.bear.cluster - Linux RHEL 8.4, POWER, 8335-GTX (power9le), Python 3.6.8
See https://gist.github.com/383182e833684da1135191378d5310b6 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 10 out of 10 (9 easyconfigs in total)
node375.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e1b58ec2c8233f5c2a415e1ca21d7f10 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 10 out of 10 (9 easyconfigs in total)
node115.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/0696345bf16c69831bf9b369ae9ce279 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in total)
sms - Linux centos linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/91a96b4337e0960df079ae48c1a9864d for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 10 out of 10 (9 easyconfigs in total)
bask-pg0309u10a - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/c4027281e02cf89d5d07131a0158214d for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
> Maybe also want to include patch for https://gcc.gnu.org/bugzilla/show_bug.cgi?id=101683 once it's done?

That's a simple (and important) enough patch, but since it should also be applied to GCC 9.3 (and perhaps 10.x), let's make that a separate PR?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (9 easyconfigs in total)
node3153.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/d79d85f2f14a4b985fcd5e77aefb1a5e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 11 out of 11 (9 easyconfigs in total)
node3537.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/e4b253529c47fbc9b8f8164846ff0dc8 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**